crates/uv-cli/src/lib.rs

```diff
@@ -3283,6 +3283,13 @@ pub struct SyncArgs {
         value_parser = parse_maybe_string,
     )]
     pub python: Option<Maybe<String>>,
+
+    /// Check if the Python environment is synchronized with the project.
```

Comment:
We should probably add another line saying like "uv will exit with a non-zero code if the environment is not up to date"